geoindicators/src/main/groovy/org/orbisgis/SpatialUnits.groovy

```diff
@@ -14,25 +14,27 @@ import org.orbisgis.processmanagerapi.IProcess
  * @param prefixName A prefix used to name the output table
  * @param datasource A connection to a database
  *
- * @return A database table name.
+ * @return A database table name and the name of the column ID
  */
 static IProcess createRSU(){
     return processFactory.create("Create reference spatial units (RSU)",
             [inputTableName: String, prefixName: String, datasource: JdbcDataSource],
-            [outputTableName : String],
+            [outputTableName : String, outputIdRsu: String],
```

Comment:
don't understand why the id column is an output value

Reply:
I added it in order to map it as an input (idColumnUp) in the createScalesRelations IProcess

Reply:
id column is a constant field so you can pass it with the same value id_rsu, id_block
Am I wrong ?

Reply:
yes but a IProcess could be used several times in a mapper and then could have a different name. Then it is necessary to map it to know where does the id used as input come from (rsu or block scale)

Reply:
ok good catch